Gemfile.lock

```diff
@@ -1,34 +1,42 @@
 PATH
   remote: .
   specs:
-    commonlit-clever-ruby (2.1.0)
+    clever-ruby (2.1.4)
```

Comment:
@geoffharcourt : I think `bundle install` will revert this back to `commonlit-clever-ruby`, which I think we want.

Reply:
Yup. I haven't automated this part yet, thanks for catching.

Reply:
Fixed in eadf45f